src/auth/src/credentials/service_account.rs

```diff
@@ -466,20 +466,18 @@ where
 
 #[allow(dead_code)]
 pub(crate) mod idtoken {
-    use std::sync::Arc;
-
-    use async_trait::async_trait;
-    use gax::error::CredentialsError;
-    use reqwest::Client;
-    use serde_json::Value;
-
     use crate::Result;
     use crate::build_errors::Error as BuilderError;
     use crate::constants::{JWT_BEARER_GRANT_TYPE, OAUTH2_TOKEN_SERVER_URL};
     use crate::credentials::idtoken::dynamic::IDTokenCredentialsProvider;
     use crate::credentials::service_account::{ServiceAccountKey, ServiceAccountTokenGenerator};
     use crate::token::{Token, TokenProvider};
     use crate::{BuildResult, credentials::idtoken::IDTokenCredentials};
+    use async_trait::async_trait;
+    use gax::error::CredentialsError;
+    use reqwest::Client;
+    use serde_json::Value;
+    use std::sync::Arc;
```

Comment:
no action required: maybe we should be standardizing our import grouping throughout our repos (nightly config for https://crates.io/crates/cargo-group-imports)